presto-main/src/main/java/com/facebook/presto/server/PrestoServer.java

```diff
@@ -121,14 +122,18 @@ public void run()
                 new EventListenerModule(),
                 new ServerMainModule(sqlParserOptions),
                 new GracefulShutdownModule(),
-                new WarningCollectorModule());
+                new WarningCollectorModule(),
+                new StrictConfigModule());
```

Comment:
would it make sense to have a BootstrapConfigModule with config for these three :

```
    private boolean quiet;
    private boolean strictConfig;
    private boolean requireExplicitBindings = true;
```

Reply:
Correct if I am wrong, these are the modules that the PrestoServer needs for startup. Do we need to add `StrictConfigModule` here since once the bootstrap is complete the server doesn't need it anymore. We are loading it explicitly in `isStrictConfig()` method to obtain the flag for bootstrapping.

Reply:
If "strict-config=true" is specified, this config will also appear in the Bootstrap object of the PrestoServer. If we do not include the module which binds the config, we'll be seeing exception that says "configuration property 'strict-config' is not used".